src/main/java/util/Utility.java

```diff
@@ -0,0 +1,22 @@
+package util;
+
+/**
+ * Class storing constants and utility methods.
+ */
+public class Utility {
+    public static final String LINE = "-------------";
+    public static final String INDENT = "    ";
+    public static final String INDENTED_LINE = Utility.INDENT + Utility.LINE;
+    public static final String NEW_LINE = System.lineSeparator();
```

Comment:
I like how you declared constants as final and used all caps for the constant names